src/ProcessManager/ProcessManager.php

```diff
@@ -91,6 +92,12 @@ private function runProcesses(array $processes)
                 if (!$process->isRunning()) {
                     --$runningProcesses;
                     ++$this->operations;
+                    if (!$process->isSuccessful()) {
```

Comment:
needs tests, or now this class is not covered (because it's hard to unit test it) but we must do it